app/bundles/CampaignBundle/Entity/ContactLimiterTrait.php

```diff
@@ -30,7 +30,7 @@ private function updateQueryFromContactLimiter($alias, DbalQueryBuilder $qb, Con
             $qb->andWhere(
                 $qb->expr()->eq("$alias.lead_id", ':contactId')
             )
-                ->setParameter('contactId', $contactId);
+                ->setParameter('contactId', $contactId, \PDO::PARAM_INT);
```

Comment:
Doesn't this cause any trouble with folks who don't have the `pdo_mysql` extension installed and instead use mysqli? I'm happy to test, but if you know it by heart then that saves me quite some time 😅